openapi3/schema_test.go

```diff
@@ -4,13 +4,13 @@ import (
 	"context"
 	"encoding/base64"
 	"encoding/json"
-	"fmt"
 	"math"
 	"reflect"
 	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v3"
```

Comment:
```suggestion
	"github.com/invopop/yaml"
```

Reply:
Well i disagree:
1. we want to use the yaml unmarshaller used in body decoder used for yaml: gopkg.in/yaml.v3 is used
2. invovop converts YAML to JSON before unmarshalling : as a consequence, we would end up with float instead of integers as we would have with yaml.v3 package